src/device-registry/routes/api-v1.js

```diff
@@ -21,10 +21,11 @@ const activityController = require("../controllers/create-activity");
 
 const headers = (req, res, next) => {
   const allowedOrigins = constants.DOMAIN_WHITELIST;
-  const origin = req.headers.origin;
-  if (allowedOrigins.includes(origin)) {
-    res.setHeader("Access-Control-Allow-Origin", origin);
-  }
+  // const origin = req.headers.origin;
+  // if (allowedOrigins.includes(origin)) {
+  //   res.setHeader("Access-Control-Allow-Origin", origin);
+  // }
+  res.setHeader("Access-Control-Allow-Origin", allowedOrigins);
```

Comment:
not checking for the origin in the headers since some SSL encrypted sites hide this information. So just explicitly sending back all the whitelisted on every request.